plugin.go

```diff
@@ -559,3 +561,133 @@ func (p *ormPlugin) generateListHandler(message *generator.Descriptor) {
 	p.P(`}`)
 	p.P()
 }
+
+/////////// For association removal during update
+
+// findAssociationKeys should return a map[childFK]parentKeyField
+func findAssociationKeys(parent *generator.Descriptor, child *generator.Descriptor, field *descriptor.FieldDescriptorProto) map[string]string {
+	// Check for gorm tags
+	parentTypeNamePb := generator.CamelCaseSlice(parent.TypeName())
+	parentTypeName := lintName(parentTypeNamePb)
+	keyMap := make(map[string]string)
+	childFields := []string{fmt.Sprintf("%sID", parentTypeName)}
+	parentFields := []string{"ID"}
+	if field.Options != nil {
+		v, err := proto.GetExtension(field.Options, gorm.E_Field)
+		if err == nil && v.(*gorm.GormFieldOptions) != nil && v.(*gorm.GormFieldOptions).Tags != nil {
```

Comment:
invert `if` and early return, all the way down.

It'd be good to store the `tag` on the initial cast instead of doing it 3 times